### PR TITLE
For CEPH_OSD_OP_COPY_GET, add a new filed: flags.

### PR DESCRIPTION
**Pull request by majianpeng**

### CODE REVIEW

File: src/osd/ReplicatedPG.cc [hunk 1]
```diff
@@ -6215,7 +6219,12 @@ void ReplicatedPG::_copy_some(ObjectContextRef obc, CopyOpRef cop)
     // it already!
     assert(cop->cursor.is_initial());
   }
-  op.copy_get(&cop->cursor, get_copy_chunk_size(),
+
+  __le32 copyget_flags = 0;
```
Comment (liewegas) [inline src/osd/ReplicatedPG.cc]: uint32_t .  the byte swabbing happens automagically when it gets assigned to the args.copy_get field.

File: src/osdc/Objecter.h [hunk 2]
```diff
@@ -694,6 +694,7 @@ struct ObjectOperation {
 
   void copy_get(object_copy_cursor_t *cursor,
 		uint64_t max,
+		__le32 copyget_flags,
```
Comment (liewegas) [inline src/osdc/Objecter.h]: uint32_t

**Comment by liewegas** [comment]
Otherwise looks good!

**Comment by liewegas** [comment]
Can you make a simple test in test/librados/tier.cc?

**Comment by majianpeng** [comment]
@liewegas . updated & please review again!

**Comment by liewegas** [comment]
Reviewed-by:

**Pull request merged**